### ISSUE TITLE
mzR dropped `mzData` format, drop or replace?

### ISSUE THREAD

**Issue by meowcat** [MassBank/RMassBank#304]
https://github.com/sneumann/mzR/blob/master/NEWS
` o Removed RAMP backend, dropping ability to read mzData `
Leading to failure on bioc build, since we still have mzData files in the RMassBankXCMS vignette:

https://github.com/MassBank/RMassBank/blob/1ea06508d1736b11070dec945631a08cee57d079/vignettes/RMassBankXCMS.Rmd#L90-L92

https://master.bioconductor.org/checkResults/3.15/bioc-LATEST/RMassBank/nebbiolo1-buildsrc.html

@sneumann, do you have these files in mzML? Or should we just drop this example?

**Comment by sneumann** [comment]
Hi, 
I converted the files in https://github.com/MassBank/RMassBankData/commit/0547193c48172c5006548f1ed425f9b2f0ca8719, 
but I have the feeling I can't push to BioC upstream for permission reasons.

I also switched filenames in the vignette in https://github.com/MassBank/RMassBank/commit/e51aa6faa8b091c53272ee62dc53a3315133b25c
But I am not going to merge anything because I am confused about branches (master, main, dev, ...) and which version is where. A PR is tried to create had far more commits that what I actually did. Can @tsufz  help ? 
Yours, Steffen
